### PR TITLE
Improvement/haste

### PR DESCRIPTION
**Pull request by bildja**
https://jira.wixpress.com/browse/WIXMP-703

### CODE REVIEW

File: scripts/npm-release.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      }
+
+      console.log('ERROR: Unable to get published version from npmjs.org', e);
+    });
+}
+
+function shouldPublish(publishedVersion, packageJSONVersion) {
+  return (
+    publishedVersion === ZERO_VERSION ||
+    (!!publishedVersion && semver.gt(packageJSONVersion, publishedVersion))
```
Comment (adavidof) [inline scripts/npm-release.js]: `Boolean(publishedVersion)`

Comment (bildja) [reply scripts/npm-release.js]: I am not even sure the `!!` is needed here, It's copy/paste from playable

File: scripts/npm-release.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
+if (!process.env.IS_BUILD_AGENT) {
```
Comment (maxaon) [inline scripts/npm-release.js]: Please add sheband `#!/usr/bin/env node`

Comment (bildja) [reply scripts/npm-release.js]: @maxaon ok, it was almost copy-pasted from playable and okidoc and it runs with `node `, but totally makes sense

**Pull request merged**